example/src/main/java/com/midtrans/sample/SampleIrisApi.java

```diff
@@ -0,0 +1,83 @@
+package com.midtrans.sample;
+
+import com.midtrans.Config;
+import com.midtrans.ConfigFactory;
+import com.midtrans.Midtrans;
+import com.midtrans.httpclient.SnapApi;
+import com.midtrans.httpclient.error.MidtransError;
+import com.midtrans.sample.data.DataMockup;
+import com.midtrans.service.MidtransSnapApi;
+import org.json.JSONObject;
+
+import java.util.Map;
+
+import static com.midtrans.sample.data.Constant.sandboxClientKey;
+import static com.midtrans.sample.data.Constant.sandboxServerKey;
+
+public class SampleIrisApi {
+
```

Comment:
Is this expected, the class say IrisApp but the implementations of this file is mostly Snap?

Reply:
Nope, it should be to iris sample implementation. Will change da